modifying_bety_schema.md

```diff
@@ -0,0 +1,70 @@
+# Modifying Bety schema
+
+Changes to the Bety schema happen through the Ruby on Rails database migration system.
+Here is a [detailed guide](http://edgeguides.rubyonrails.org/active_record_migrations.html) on database migration for Ruby on Rails.
+What follows is a lightning guide to the essential steps.
+
+1. If you have not done so already, `git clone` the Bety repository. If you are on the VM, it should already be present in `~/bety`.
+
+2. `cd` into the `bety` repository and update to the latest version with `git pull`.
+
+3. Make sure all Ruby gems are up to date. Running `bundle check` should do a quick check.
+   If anything is out of date, update with `bundle install --path vendor/bundle`.
+   (Adding the `--path` argument is strongly recommended to isolate these gems from the system install.)
+   If this throws an error, see [Troubleshooting](#modifying-bety-troubleshooting) below.
+
+4. Check the `config/database.yml` file to make sure that database parameters are set correctly.
+   If you have multiple instances of Bety installed (e.g. `development`, `production`), adjust parameters accordingly.
+   You can name each database environment whatever you want, but this guide assumes you are working with the `production` configuration (the default on the VM).
+   The default VM configuration should look like this:
+
+```
+production:
+  adapter: postgis
+  encoding: utf-8
+  reconnect: false
+  database: bety
+  pool: 5
+  username: bety
+  password: bety
+```
+
+5. Update the Bety database itself with the latest migrations by running `bundle exec rake db:migrate RAILS_ENV=production` (if you are using a different BETY instance, change `production` to whatever you are working on).
+   If this command exits silently, then your bety instance is up to date.
+   If it throws an error, see [Troubleshooting](#modifying-bety-troubleshooting) below.
+
+6. Modifying the database involves adding new Ruby scripts to the `db/migrate` directory.
+   In a nutshell, each migration describes a database revision (`up` -- how to make the change) and its inverse (`down` -- how to undo the change).
+   There are lots of examples of previous migrations in this directory that can be used for reference.
+   Database operations can be performed by passing SQL queries directly to an `execute %q{<YOUR QUERY HERE>}` statement, or through Ruby helper commands like `change_column`.
+   Although you can name migrations whatever you want, because they are run in numerical-alphabetical order, the convention is to prefix your script with the current year, month, day, hour, minute, and second (you can generate this string by running `date +%Y%m%d%H%M%S` on the command line).
```

Comment:
another option is that you can use `bundle exec rails generate migration ChangeParameterListToText`. This will generate the migration for you with the current datetime.

Reply:
Didn't know about that, but that's really cool! I'll add it to this.